docs/vsphere-golden-image.md

```diff
@@ -0,0 +1,191 @@
+# Creating the vSphere Windows VM Golden Image
+
+This guide describes the thought process of creating a Windows virtual machine base image (the Golden Image) in vSphere.
+
+## 1. Select a compatible Windows Server version
+
+Currently, the Windows Machine Config Operator (WMCO) stable version only supports Windows Server version 1909 
+(build 18363.\*) Semi-Annual Channel, that includes the patch [KB4565351](https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351), 
+required to get the operating system feature that allows setting the [hybrid OVN Kubernetes networking with a 
+custom VXLAN port](setup-hybrid-OVNKubernetes-cluster.md#vSphere).
```

Comment:
The point of this PR is adding support for 2004. So:
```
Windows Machine Config Operator (WMCO) only supports Windows Server Semi-Annual Channel (SAC): Windows Server 2004, that includes...
```
Lets be consistent with the verbiage used in the OpenShift docs and in https://github.com/openshift/windows-machine-config-operator/pull/476/